` [ERROR] Needs systemd (systemctl --user)` (observed in rawhide)

> This PR kind of works, but rootless setup has failed:
> 
> ```
> ❯ l shell fedora-rawhide
> [systemd]
> Failed Units: 1
>   cloud-init-main.service
> 
> [jan@lima-fedora-rawhide lima]$ sudo journalctl -u cloud-init-main.service
> ...
> Mar 16 18:09:00 lima-fedora-rawhide cloud-init[907]: + sudo -iu jan XDG_RUNTIME_DIR=/run/user/501 PATH=/mnt/lima-cidata/util:/usr/local/bin:/usr/bin containerd-rootless-setuptool.sh install
> Mar 16 18:09:00 lima-fedora-rawhide sudo[2063]:     root : PWD=/home/jan.guest ; USER=jan ; ENV=XDG_RUNTIME_DIR=/run/user/501 PATH=/mnt/lima-cidata/util:/usr/local/bin:/
> usr/bin ; COMMAND=/bin/bash --login -c 'containerd-rootless-setuptool\\.sh install'
> Mar 16 18:09:01 lima-fedora-rawhide cloud-init[907]: [ERROR] Needs systemd (systemctl --user)
> Mar 16 18:09:01 lima-fedora-rawhide cloud-init[907]: LIMA 2026-03-16T18:09:01-07:00| WARNING: Failed to execute /mnt/lima-cidata/boot.Linux/40-install-containerd.sh
> Mar 16 18:09:01 lima-fedora-rawhide cloud-init[907]: LIMA 2026-03-16T18:09:01-07:00| Exiting with code 1
> ```

_Originally posted by @jandubois in https://github.com/lima-vm/lima/issues/4738#issuecomment-4071678252_

--- a/hack/update-template-fedora.sh
+++ b/hack/update-template-fedora.sh
@@ -36,8 +36,8 @@ Examples:
   Update the Fedora Linux image location in templates/**.yaml:
   $ $(basename "${BASH_SOURCE[0]}") templates/**.yaml
 
-  Update the Fedora Linux image location to version 41 in ~/.lima/fedora/lima.yaml:
-  $ $(basename "${BASH_SOURCE[0]}") --version 41 ~/.lima/fedora/lima.yaml
+  Update the Fedora Linux image location to version 44 in ~/.lima/fedora/lima.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") --version 44 ~/.lima/fedora/lima.yaml
   $ limactl factory-reset fedora
 
 Flags:

--- a/pkg/cidata/cidata.TEMPLATE.d/boot.Linux/40-install-containerd.sh
+++ b/pkg/cidata/cidata.TEMPLATE.d/boot.Linux/40-install-containerd.sh
@@ -120,20 +120,20 @@ EOF
 			setenforce 0
 		fi
 		if [ "$(sudo -iu "${LIMA_CIDATA_USER}" sh -ec 'systemctl --user show --property=RefuseManualStart --value dbus')" != "yes" ]; then
-			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" systemctl --user enable --now dbus
+			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" systemctl --user enable --now dbus.socket
 		fi
-		sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" containerd-rootless-setuptool.sh install
-		sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" \
+		sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" "TERM=${TERM}" containerd-rootless-setuptool.sh install
+		sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" "TERM=${TERM}" \
 			"CONTAINERD_NAMESPACE=${CONTAINERD_NAMESPACE}" "CONTAINERD_SNAPSHOTTER=${CONTAINERD_SNAPSHOTTER}" \
 			containerd-rootless-setuptool.sh install-buildkit-containerd
 
 		# $CONTAINERD_SNAPSHOTTER is configured in 20-rootless-base.sh, when the guest kernel is < 5.13, or the instance was created with Lima < 0.9.0.
 		if [ "$(sudo -iu "${LIMA_CIDATA_USER}" sh -ec 'echo $CONTAINERD_SNAPSHOTTER')" = "fuse-overlayfs" ]; then
-			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" containerd-rootless-setuptool.sh install-fuse-overlayfs
+			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" "TERM=${TERM}" containerd-rootless-setuptool.sh install-fuse-overlayfs
 		fi
 
 		if compare_version.sh "$(uname -r)" -ge "5.13"; then
-			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" containerd-rootless-setuptool.sh install-stargz
+			sudo -iu "${LIMA_CIDATA_USER}" "XDG_RUNTIME_DIR=/run/user/${LIMA_CIDATA_UID}" "PATH=${PATH}" "TERM=${TERM}" containerd-rootless-setuptool.sh install-stargz
 		else
 			echo >&2 "WARNING: the guest kernel seems older than 5.13. Skipping installing rootless stargz."
 		fi

--- a/templates/README.md
+++ b/templates/README.md
@@ -48,7 +48,8 @@ lima
 - [`debian-13`](./debian-13.yaml), `debian`: ⭐Debian GNU/Linux 13 (Trixie)
 - [`fedora-41`](./fedora-41.yaml): Fedora 41
 - [`fedora-42`](./fedora-42.yaml): Fedora 42
-- [`fedora-43`](./fedora-43.yaml), `fedora`: ⭐Fedora 43
+- [`fedora-43`](./fedora-43.yaml), Fedora 43
+- [`fedora-44`](./fedora-44.yaml), `fedora`: ⭐Fedora 44
 - [`opensuse-leap-15`](./opensuse-leap-15.yaml): openSUSE Leap 15
 - [`opensuse-leap-16`](./opensuse-leap-16.yaml), `opensuse-leap`, `opensuse`: ⭐openSUSE Leap 16
 - [`oraclelinux-8`](./oraclelinux-8.yaml): Oracle Linux 8

--- a/templates/_images/fedora-44.yaml
+++ b/templates/_images/fedora-44.yaml
@@ -1,0 +1,14 @@
+images:
+- location: "https://download.fedoraproject.org/pub/fedora/linux/releases/44/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-44-1.7.x86_64.qcow2"
+  arch: "x86_64"
+  digest: "sha256:28680fe5b371a5a82ebf43a31926e086a168e59949d03969c5093e7071f90b7f"
+
+- location: "https://download.fedoraproject.org/pub/fedora/linux/releases/44/Cloud/aarch64/images/Fedora-Cloud-Base-Generic-44-1.7.aarch64.qcow2"
+  arch: aarch64
+  digest: "sha256:55c60a3b80d3616a08705afd0459e75fe9f03c54aba7a46e4002a41a72fa0d5b"
+
+# No RISC-V release yet for Fedora 44: https://download.fedoraproject.org/pub/alt/risc-v/release/
+
+# # NOTE: Intel Mac with macOS prior to 15.5 requires setting vmType to qemu
+# # https://github.com/lima-vm/lima/issues/3334
+# vmType: qemu

--- a/templates/_images/fedora.yaml
+++ b/templates/_images/fedora.yaml
@@ -1,1 +1,1 @@
-fedora-43.yaml
+fedora-44.yaml

--- a/templates/fedora-44.yaml
+++ b/templates/fedora-44.yaml
@@ -1,0 +1,5 @@
+minimumLimaVersion: 2.0.0
+
+base:
+- template:_images/fedora-44
+- template:_default/mounts

--- a/templates/fedora.yaml
+++ b/templates/fedora.yaml
@@ -1,1 +1,1 @@
-fedora-43.yaml
+fedora-44.yaml

